core/modules/node/node.theme.inc

```diff
@@ -4,6 +4,34 @@
  * Theme functions for the Node module.
  */
 
+/**
+ * Returns HTML for node_preview_banner_form().
+ *
+ * @param $variables
+ *   An associative array containing:
+ *   - form: A render element representing the form.
+ *
+ * @see node_preview_banner_form()
+ * @ingroup themeable
+ */
+function theme_node_preview_banner_form($variables) {
+  $form = $variables['form'];
+
+  $output = '<div class="node-node-preview-banner-form">';
+  $output .= '<div class="node-node-preview-banner-form-left">';
+  $output .= backdrop_render($form['preview_node_submit']);
+  $output .= backdrop_render($form['backlink']);
+  $output .= '</div>';
+  $output .= '<div class="node-node-preview-banner-form-right">';
```

Comment:
Rename to `node-node-preview-banner-form-submit` and `node-node-preview-banner-form-view-mode`.